### PR TITLE
Reflection probes documentation

### PR DESCRIPTION
**Pull request by Sanakan8472**

### CODE REVIEW

File: pages/docs/graphics/lighting/reflection-probe-components.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+![Box vs sphere reflection probe](media/reflection-probes.jpg)
+
+By default, each probe captures everything in its radius. In many cases it makes sense to [tag](../../projects/tags.md) the `ExcludeTags` with `SkyLight` though so that the resulting cubemap is transparent where no geometry was rendered. This allows for the cubemaps to be dynamically composited at runtime. The benefits of this is that the skylight can change dynamically and won't be baked into the probe and it allows for parallax between e.g. a box probe and the sky light fallback reflection.
+
+The captured reflection is only visible in the probe's influence volume. For sphere probes this is defined by their radius and for box probes by their extents.
+
+At runtime, all probes are sorted by their volume and the smallest probe is sampled first. If the reflection is transparent in the cubemap, then next bigger cubemap that influences the geometry is sampled next. This continues until we hit an opaque pixel in a cubemap or if the fallback reflection of the * [sky light component](sky-light-component.md) is reached.
```
Comment (jankrassnigg) [inline pages/docs/graphics/lighting/reflection-probe-components.md:16]: unnecessary *

File: pages/docs/graphics/lighting/reflection-probe-components.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Sphere reflection component:
+
+* `Radius`: The influence range of the probe. The reflection is only visible on geometry intersecting this radius.
+* `Falloff`: Percentage of the radius that is smoothly blended into other probes.
+
+Box reflection component:
+
+* `Extents`: The extents of the box projection. The cubemap will be projected to this box.
+* `InfluenceScale`, * `InfluenceShift`: The influence volume can be smaller than the projected volume (extents) of the probe. This can be useful if your have e.g. a long corridor that you want to place multiple probes in. Each will have the same projection but a different part of the projection volume will be set as the influence volume centred around a probe capture offset.
```
Comment (jankrassnigg) [inline pages/docs/graphics/lighting/reflection-probe-components.md:30]: unnecessary * in the text

Comment (jankrassnigg) [inline pages/docs/graphics/lighting/reflection-probe-components.md:30]: "if you have" - not "if your have"

File: pages/docs/graphics/lighting/reflection-probe-components.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* `PositiveFalloff`, * `NegativeFalloff`: Percentage of the influence volume in each direction that is smoothly blended into other probes. The falloff is defined for each face of the box.
+
+Common properties for both sphere and box reflection probes that describe how the probe is captured:
+
+* `ReflectionProbeMode`: `Dynamic` makes the skylight update continuously. `Static` will only update once at the start.
+* `IncludeTags`, `ExcludeTags`: These [tags](../../projects/tags.md) define which objects in the scene are used to compute the skylight. Make sure that the object that renders your [sky](../../effects/sky.md) has this include tag set. This is the same mechanism as used in the [camera component](../camera-component.md).
```
Comment (jankrassnigg) [inline pages/docs/graphics/lighting/reflection-probe-components.md:36]: I think this was copied from the sky component? Needs to be adjusted.

File: pages/docs/graphics/lighting/reflection-probe-components.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+* `NearPlane`, `FarPlane`: Camera settings used when the lighting is captured from the scene. If `NearPlane` is set to `Auto`, a value is computed automatically from the `FarPlane`.
+* `CaptureOffset`: The capture offset allows for the capture position of the probe to be decoupled from the game object position.
+* `ShowDebugInfo`: If enabled, a sphere with a preview of the sky image is rendered at the position of the skylight object. Use this to check whether all desired objects contribute to the skylight.
```
Comment (jankrassnigg) [inline pages/docs/graphics/lighting/reflection-probe-components.md:39]: sky

File: pages/docs/graphics/lighting/reflection-probe-components.md (hunk 3 above)
Comment (Sanakan8472) [reply pages/docs/graphics/lighting/reflection-probe-components.md]: Fixed. Also made ExcludeTags to default to SkyLight so placed probes automatically composite with the fallback probe.

File: pages/docs/graphics/lighting/reflection-probe-components.md (hunk 2 above)
Comment (Sanakan8472) [reply pages/docs/graphics/lighting/reflection-probe-components.md]: Fixed.

File: pages/docs/graphics/lighting/reflection-probe-components.md (hunk 4 above)
Comment (Sanakan8472) [reply pages/docs/graphics/lighting/reflection-probe-components.md]: Done. Also added a comment regarding the stack of spheres of increasing roughness.

**Pull request merged**